__test__/integration/loan_offer/runners/create.ts

```diff
@@ -0,0 +1,19 @@
+import { Dharma } from "../../../../src";
+
+import { LoanOffer } from "../../../../src/types";
+
+export async function testCreate(dharma: Dharma, params: any) {
+    describe("passing valid params", () => {
+        let loanOffer: LoanOffer;
+
+        beforeAll(async () => {
+            loanOffer = await LoanOffer.create(dharma, params) as LoanOffer;
```

Comment:
so this should be: `LoanOffer.create<LoanOffer>(dharam, params)`

Reply:
oh of course! who could forget?